libs/CMakeLists.txt

```diff
@@ -3,4 +3,7 @@ add_library(toylib STATIC
                 toylib.hpp
                 toylib.cpp
 )
-target_include_directories(toylib PUBLIC .)
\ No newline at end of file
+target_include_directories(toylib PUBLIC $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/libs>)
+set_target_properties(toylib PROPERTIES PUBLIC_HEADER "toylib.hpp")
+
+install(TARGETS toylib EXPORT atlasutils PUBLIC_HEADER LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/)
```

Comment:
installing the lib so that other dependent projects can use it by loading the corresponding cmake exported target